doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -232,10 +232,21 @@ Or::
 The best way to deal with a full cluster is to add new ``ceph-osds``, allowing
 the cluster to redistribute data to the newly available storage.
 
-If you cannot start an OSD because it is full, you may delete some data by deleting
-some placement group directories in the full OSD.
+If you cannot add a new OSD, you can do the following:
```

Comment:
Just deleting PGs is not possible, as it gets immediately retransferred to the OSD. The current solution suggested is therefore not working. I doubt that deleting PGs has ever been the right solution for this task. I have not found one source of documentation that could convince me otherwise. I'd very welcome someone's else opinion on this.

I would change "if you can not add a new OSD" to: "If you need to regain access to the cluster right away, without adding a new OSD".